spec/support/shared_examples/activity_notifications.rb

```diff
@@ -10,8 +10,8 @@
 
     it 'for each user' do
       run_action(subject)
```

Comment:
will an inactive user get notifications and what is expected in that regard?

Reply:
Yes, good point. I'm not sure what the behavior should be in that case. As we discussed in our pairing session, the notifications are only internal to the site and are not sent by any messenger or email so they would not be able to see them unless someone were to reactivate them. I'm not sure exactly what the client would expect to occur in that case. I could see reasons for them to want the user to see, or not see, the notifications.

I'll ask @drjakob about it when I next speak with him.